components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -1111,12 +1111,18 @@ public class MonitoredItemWrapper {
                         }
                     }
 
+                    List<SimpleAttributeOperand> addedSelectClauses = new List<SimpleAttributeOperand>();
```

Comment:
Could you add a comment about this so it's clear what we are doing? And perhaps call it internalSelectClauses.

Reply:
Good point. Fixed.